shared/install/macos.js

```diff
@@ -161,7 +161,10 @@ const isInstalled = async function () {
  */
 const getExecOptions = async function () {
   return {
-    cwd: resolve(__dirname, '../../MacOSATDriverServer/Build/Debug'),
+    cwd: resolve(
+      __dirname,
+      '../../packages/macos-at-driver-server/MacOSATDriverServer/Build/Debug',
+    ),
```

Comment:
[The entire `shared` directory is copied into the package directory whenever we package the module for publication](https://github.com/bocoup/at-driver-servers/blob/e75a2e9952b79022984b5169fd05015df366a0ab/packages/macos-at-driver-server/package.json#L16), so the path as expressed on the `main` branch is actually correct. For local testing, you can run `npm run prepack` from the `packages/macos-at-driver-server` directory, and then use the `./shared/bin/at-driver` executable from there.

(Won't it be nice when we remove Windows support and get rid of this indirection?)